MYCONTRIBUTION.md

```diff
@@ -0,0 +1 @@
+```Clean up PR descriptions by removing comments #2017```
```

Comment:
Could we remove `MYCONTRIBUTION.md` file? Apache fury use `git history` to credit the contributors